<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KUSBLFTOASCClass</class>
 <widget class="QWidget" name="KUSBLFTOASCClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KUSBLFTOASC</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_blf">
       <property name="minimumSize">
        <size>
         <width>385</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>395</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_sleBlf">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>选择blf文件</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit_asc">
       <property name="minimumSize">
        <size>
         <width>385</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>395</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_selAsc">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>选择保存路径</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="lineEdit_filter"/>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_filter">
       <property name="text">
        <string>筛选ID(分号隔开)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_converter">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>转换</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="KUSBLFTOASC.qrc"/>
 </resources>
 <connections/>
</ui>
